doc/api/magic-plot.md

```diff
@@ -0,0 +1,154 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# `%sqlplot`
+
+```{versionadded} 0.5.2
+```
+
+
+```{note}
+`%sqlplot` requires `matplotlib`: `pip install matplotlib` and this example requires
+duckdb-engine: `pip install duckdb-engine`
```

Comment:
I get why duckdb, maybe we can add some menu to pick between dbs? like sqllite, ddb, snowflake? Kind of what we did for cloud providers.

Reply:
SQLite is not compatible with this module. the problem is ensuring that those examples run. we'd need to have running instances of each dynamo and snowflake. it's an overkill at this point.

Reply:
We didn't validate it in soopervisor.
I think we can skip this for now and discuss.